x-pack/legacy/plugins/canvas/public/style/main.scss

```diff
@@ -40,22 +39,22 @@ $canvasElementCardWidth: 210px;
   border-top: $euiBorderThin;
 }
 
-#canvas-app {
+#canvas-app { // sass-lint:disable-line no-ids
   overflow-y: hidden;
 
   .window-error {
     position: absolute;
     bottom: 10px;
     right: 10px;
-    background-color: red;
-    color: white;
+    background-color: $euiColorDanger;
+    color: $euiColorEmptyShade;
```

Comment:
This needs to be white in both light and dark mode, so you'll need to use `$euiColorGhost` here as it is always #FFF regardless of theme whereas `$euiColorEmptyShade` will flip.